tests/daily_scraper/test_daily_scraper.py

```diff
@@ -305,14 +316,36 @@ def test_scraper(self):
         """Testing various download scenarios for DailyScraper"""
 
         for entry in tests:
-            scraper = DailyScraper(directory=self.temp_dir, base_url=self.wdir,
+            scraper = DailyScraper(destination=self.temp_dir, base_url=self.wdir,
                                    version=None, log_level='ERROR',
                                    **entry['args'])
-            expected_target = os.path.join(self.temp_dir, entry['target'])
-            self.assertEqual(scraper.target, expected_target)
             self.assertEqual(urllib.unquote(scraper.final_url),
                 urljoin(self.wdir, entry['target_url']))
 
+    def test_destination(self):
```

Comment:
There is no need to test this here. `target` and `destination` are properties of the `BaseScraper` class and should only be tested there.